src/components/Placeholder/Placeholder.tsx

```diff
@@ -0,0 +1,75 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core"
+import React, { useContext } from "react"
```

Comment:
To be consistent with other code in Gatsby Interface: 
```suggestion
import * as React from "react"
```